cypress/integration/WidgetsForEnrollmentPages/WidgetEventComment/index.js

```diff
@@ -2,6 +2,14 @@ Then('the enrollment widget should be loaded', () => {
     cy.contains('The enrollment event data could not be found').should('not.exist');
 });
 
+When('you click edit mode', () => {
+    cy.get('[data-test="widget-enrollment-event"]').within(() => {
+        cy.get('[data-test="dhis2-uicore-button"]')
+            .contains('Edit event')
+            .click();
+        cy.wait(100);
```

Comment:
Hey, 
Why is there a need to wait for 100 ms? Maybe you can wait for some text to appear on the page rather than have a fixed waiting time 🤔. 
The similar step is used in [EnrollmentEditEventPageForm](https://github.com/dhis2/capture-app/blob/master/cypress/integration/EnrollmentEditEventPage/EnrollmentEditEventPageForm/index.js#L9) and it works without `cy.wait`. 
Thanks!

Reply:
hey it needs to wait for the widget to be loaded and redux data added before accessing the form, I tried to wait for a text but doesn't work so keep it like that

Reply:
@jasminenguyennn can you have a second look at this one. I want us to try very hard to avoid having waiting statements in the code, we introduce an unreliable test here that might fail if the test execution takes longer than usual.

Reply:
@Bekkalizer sure. I updated it, if we are not rushing for merging as QA wanted.